### PR TITLE
split the title check from the e2e test

### PR DESCRIPTION
**Pull request by jianzhangbjz**
As title, split the title check from the e2e test.

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
@petr-muller Could you help approve it? Thanks!

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -190,7 +190,7 @@ approve:
   - openshift-priv/openshift-tests-private
   - openshift/openshift-tests
   - openshift/openshift-tests-private
-  require_self_approval: true
+  require_self_approval: false
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:193]: This is unrelated to the the title of the PR and has no justification. Do you know what this does?

Also, I don't think you should be changing approval settings for `openshift/openshift-tests`.

Comment (jianzhangbjz) [reply core-services/prow/02_config/_plugins.yaml:193]: Yes, I know. I hope the OWNERS don't need to approve their PR explicitly. In other words, their PR can be approved automatically.
> Also, I don't think you should be changing approval settings for openshift/openshift-tests.

Why? As you know, only QE is using this repo now.

Comment (jianzhangbjz) [reply core-services/prow/02_config/_plugins.yaml]: Anyway, I removed this setting change.
